package.json

```diff
@@ -101,7 +101,7 @@
     "@lavamoat/preinstall-always-fail": "^1.0.0",
     "@material-ui/core": "^4.11.0",
     "@metamask/contract-metadata": "^1.26.0",
-    "@metamask/controllers": "^10.0.0",
+    "@metamask/controllers": "https://github.com/MetaMask/controllers#2ad3f2d1e5a235e5d6bdc47851f151476f2efa80",
```

Comment:
Is this just a temporary solution?

Reply:
Yes, so that we're unblocked until the real package gets published.